### PR TITLE
doc/radosgw/adminops: add PHP binding libraries 

### PR DESCRIPTION
**Pull request by lbausch**
This adds some PHP binding libraries for the Admin OPS API. Disclaimer: I maintain one of them (`lbausch/php-ceph-radosgw-admin`).

## Checklist
- [x] References tracker ticket
- [x] Updates documentation if necessary
- [x] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/radosgw/adminops.rst [hunk 1]
```diff
@@ -1976,9 +1976,14 @@ Binding libraries
  - `QuentinPerez/go-radosgw`_
```
Comment (anthonyeleven) [inline doc/radosgw/adminops.rst:1977]: Should we add `https://github.com/ceph/go-ceph` while we're here?

**Review by anthonyeleven — CHANGES_REQUESTED** [review]

File: doc/radosgw/adminops.rst (hunk 1 above)
Comment (lbausch) [reply doc/radosgw/adminops.rst:1977]: @anthonyeleven I guess this is addressed in #43085 ?

Comment (anthonyeleven) [reply doc/radosgw/adminops.rst:1977]: Aye, I had not seen that PR yet when I commented.

**Review by anthonyeleven — APPROVED** [review]

**Comment by anthonyeleven** [comment]
There are some check complaints that may be a function of when this PR was created.  I'm going to wave them through and merged.

**Pull request merged**